RELEASE.md

```diff
@@ -1,5 +1,13 @@
 ## Release Note 🍻
 
+- Set default resource for containers
```

Comment:
This is a new feature? I didn't see a PR for this

Reply:
This is not a feature,just changes.

Reply:
Plz show code for me